src/platform/elements/date-picker/DatePickerInput.ts

```diff
@@ -45,20 +47,30 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
   @Input() placeholder: string;
   @Input() maskOptions: any;
   @Input() format: string;
+  @Input() textMaskEnabled: boolean;
```

Comment:
Can default to true, then you don't need to check for undefined below

Reply:
nvm, I see how you are doing it, maybe flip it? `disabledTextMask: boolean = false` then down below itll be `!!field.disableTextMask`

Reply:
You were right.  I wanted it to just default to true. I just set the default.  It works just fine.